### PR TITLE
correct base64 wide keyword

### PR DESCRIPTION
**Pull request by wesinator**
https://yara.readthedocs.io/en/latest/writingrules.html#base64-strings

### CODE REVIEW

**Comment by infosec-intern** [comment]
Thanks for catching that @wesinator! Sorry it took so long. I haven't been active on GitHub lately outside of work. Out of curiosity, are you using this project much?

**Comment by wesinator** [comment]
> Thanks for catching that @wesinator! Sorry it took so long. I haven't been active on GitHub lately outside of work. Out of curiosity, are you using this project much?

I mean I use the VScode yara grammar, yes
Not sure if it's using this one or the old repo, seems this one is updated.

p.s. could you look at archiving https://github.com/infosec-intern/textmate-yara if it's not used ? 

Thanks

**Comment by infosec-intern** [comment]
Ah, gotcha. The older textmate-yara repo is actually the one still being pushed to the VSCode marketplace while I'm fixing bugs in this one and trying to get it to install properly on all OSes without any extra hassle. My plan in the future is to push this out as a new extension and to start notifying users of the textmate-yara extension to move to this one. Given how differently this one is structured from the old extension, I'm expecting a lot of bugs, and I don't want to archive the old one until I'm confident this one works 99% of the time.

**TL;DR** textmate-yara is still the one everyone uses, but I will archive it when that is not the case, and I really appreciate you bringing up these issues (and especially submitting PRs for them).
